projects/ngrx.io/content/guide/signals/signal-store/private-store-members.md

```diff
@@ -0,0 +1,67 @@
+# Private Store Members
+
+SignalStore allows defining private members that cannot be accessed from outside the store by using the `_` prefix.
```

Comment:
Should we mention a compilation error is displayed when trying to access them?

Reply:
This was shown in the example: https://github.com/ngrx/platform/pull/4447/files#diff-9dd7a2eeb44718a61b7b0b462918eded72e902854e254161eb35629869d6f8f0R56